docusaurus/docs/create-a-plugin/extend-a-plugin/instrument-a-backend-plugin.md

```diff
@@ -0,0 +1,421 @@
+---
+id: instrument-a-backend-plugin
+title: Instrument a backend plugin
+description: How to instrument backend plugins using logs, metrics and traces.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - instrument
+  - instrumentation
+  - logs
+  - metrics
+  - distributed tracing
+  - tracing
+  - backend
+  - back-end
+---
+
+# Instrument a backend plugin
+
+Guidance, conventions and best practices for instrumenting backend plugins using logs, metrics and traces to make it easier to diagnose and resolve issues for both plugin developers and Grafana operators.
+
+## Logs
+
+Logs are files that record events, warnings and errors as they occur within a software environment. Most logs include contextual information, such as the time an event occurred and which user or endpoint was associated with it.
+
+### Implement logging in your plugin
+
+Using the global logger, `backend.Logger`, from the [backend package](https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go/backend) works everywhere and for most use cases.
+
+**Example:**
+
+```go
+package plugin
+
+import (
+    "errors"
+
+    "github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+func main() {
+    backend.Logger.Debug("Debug msg", "someID", 1)
+    backend.Logger.Info("Info msg", "queryType", "default")
+    backend.Logger.Warning("Warning msg", "key", "value")
+    backend.Logger.Error("Error msg", "error", errors.New("An error occurred"))
+}
+```
+
+:::note
+
+The `backend.Logger` is a convenient wrapper over `log.DefaultLogger` from the [log package](https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go/backend/log) which you also can use to access the global logger.
+
+:::
+
+#### Reuse logger with certain key/value pairs
```

Comment:
This isn't strictly related to this PR, but when rendered in docusaurus, to me this H4 header looks like normal text. It took me a few seconds to realize it was a new paragraph:
![immagine](https://github.com/grafana/plugin-tools/assets/16373015/015d4e7f-10d5-499f-81cc-5fb56ed4d036)

Maybe it's just me? cc @josmperez

Reply:
Sounds good! :+1: